config/sync/node.type.magazine.yml

```diff
@@ -9,7 +9,7 @@ third_party_settings:
     available_menus:
       - main
     parent: 'main:'
-name: Magazine
+name: Magasin
```

Comment:
This should probably be handled by translating the content type.

Reply:
true, jeg gør det lige i et snuptag